### DISCUSSION TITLE
Why is my DevToys version 1.0.14.0?

### DISCUSSION THREAD

**Discussion by bellascalzi1** [DevToys-app/DevToys#1248] [Q&A] [answered]
Hi, I was just wondering if the latest version of DevToys 1.x is 1.0.14.0? That is what the version is saying in Devtoys on my laptop, but I noticed that the latest 1.x version in the Releases section on GitHub is 1.0.13.0, and the release notes in the Microsoft store only go up to 1.0.13.0 https://apps.microsoft.com/detail/9pgcv4v3bk4w?hl=en-US&gl=US

I did notice that there is a 1.0.14.0 branch, It may be that the developer just hasn't updated the releases section but I just thought I would ask.

Thanks!

**Comment by veler** [comment, marked as answer]
Hello,
Thanks for checking out. The only purpose of 1.0.14.0 was to display this notification, and that's it. There's no other change in this version.
![image](https://github.com/DevToys-app/DevToys/assets/3747805/d17792e9-675b-4053-ae70-b9aa7fa72186)

DevToys 1.0 won't receive further update. We're now entirely working on DevToys 2.0, which is currently in beta.

You can try it out here: https://devtoys.app/download
You can also see the release details here: https://github.com/DevToys-app/DevToys/releases and on our blog: https://devtoys.app/blog/announcing-devtoys-2.0-preview

Thanks

**Comment by bellascalzi1** [comment]
Cool, thank you so much for the quick reply! I will check out v2.0 then!
